routes/server.go

```diff
@@ -295,6 +295,10 @@ const (
 	// associations.go
 	RoutePathUserAssociations = "/api/v0/user-associations"
 	RoutePathPostAssociations = "/api/v0/post-associations"
+
+	// snapshot.go
+	RoutePathGetSnapshotEpochMetadata = "/api/v0/get-snapshot-epoch-metadata"
+	RoutePathGetStateChecksum         = "/api/v0/get-state-checksum"
```

Comment:
since these are really `GET` routes, we don't need to include `get` in the route.